pywbemtools/pywbemcli/_cmd_connection.py

```diff
@@ -251,21 +272,26 @@ def connection_select(context, name):
                      default='all'))
 @add_options(verify_option)
 @click.pass_obj
-def connection_add(context, **options):
+def connection_add(context, name, **options):
     """
     Add a new WBEM connection definition from specified options.
 
     Create a new WBEM connection definition in the connections file from
     the specified options. A connection definition with that name must not
     yet exist.
 
-    The current connection remains unchanged by this command.
+    The NAME argument MUST exist. It define the server uri
+    and the unique name under which this server connection information
+    will be stored. A --server or --mock-server must exist because they
+    define the server. All other properties are optional.
```

Comment:
On "A --server or --mock-server must exist because they define the server.":

1. Again the wording "exist"
2. Missing word "option" after the option names
3. The --name option also works.

I suggest to change the quoted sentence to:

"One of the --server, --mock-server or --name options must be specified to define the WBEM server for the new connection definition."

Reply:
DONE